Mage/src/main/java/mage/filter/predicate/DynamicIntComparePredicate.java

```diff
@@ -0,0 +1,42 @@
+
+package mage.filter.predicate;
+
+import mage.MageObject;
+import mage.abilities.Ability;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.effects.Effect;
+import mage.constants.ComparisonType;
+import mage.game.Game;
+
+/**
+ *
+ * @author North
```

Comment:
Is this correct?

Reply:
Ah sorry, let me go ahead and fix the documentation